lib/notifications/db/detour.ex

```diff
@@ -33,4 +43,60 @@ defmodule Notifications.Db.Detour do
       :status
     ])
   end
+
+  defmodule Queries do
```

Comment:
This module feels worth testing explicitly, because I imagine we're going to re-use it a lot.

Those tests could probably be added as a fast-follow, but I wouldn't hate to see them added directly to this PR.

Reply:
Right now I feel comfortable with the level of testing done in the `notifications_test`, because every line of code in this is required and tested (IIRC, I could've missed something).

I'd be open to writing a module to test it specifically, but I don't find it very high priority and I'm having trouble thinking of what that would look like _today_. Maybe revisiting this sometime in the next week I'll be in a better space to take that on.

Alternatively, I'd be happy to pair on it!

Reply:
Done!